src/components/Submit.tsx

```diff
@@ -0,0 +1,88 @@
+import { addDoc } from "firebase/firestore";
```

Comment:
Directly using `firestore` in components is not recommended. Would suggest creating a service instead (`services/product-service.tsx`) and add functionality there (eg - `createProduct(product Product)`).